src/components/views/dialogs/AddExistingToSpaceDialog.tsx

```diff
@@ -197,12 +197,13 @@ export const AddExistingToSpace: React.FC<IAddExistingToSpaceProps> = ({
                 setProgress(i => i + 1);
             } catch (e) {
                 logger.error("Failed to add rooms to space", e);
-                setError(error = e);
+                error = e;
                 break;
             }
         }
 
         if (!error) {
+            setError(error);
```

Comment:
Shouldn't this go under an `if (error)`?

Reply:
Err yes, good spot!